src/components/containers/country.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+import Cookies from 'js-cookie'
+import { isEuCountry } from 'common/country-base'
+import { BinarySocketBase } from 'common/websocket/socket_base'
+
+const handleEu = (setVisible, to) => is_eu_country => {
+    switch (to) {
+        case 'eu':
+            setVisible(is_eu_country)
+            break
+        case 'non-eu':
+            setVisible(!is_eu_country)
+            break
+        default:
+            break
+    }
+}
+
+const Country = ({ children, to }) => {
```

Comment:
```suggestion
const Location = ({ children, to }) => {
```

location is probably a better name since EU is not a country